README_DEVELOPERS.md

```diff
@@ -43,3 +43,38 @@ npm run test -- -- -t react
 ## Replace etalon
 
 To replace etalon image just remove it from `packages/devextreme-cli/testing/__tests__/__image_snapshots__` folder and run tests again
+
+# Modifying Application Templates
+
+If you need to modify an application template, we recommend that you create a real application based on this template and modify this application instead. This is easier because a real application can be run, and you can see how your modifications affect it, which would not be possible if you were to modify the template directly.
```

Comment:
I suppose it's unclear here that the real application should not be created manually. It will by created in `testing/sandbox` folder by the testing environment.

Reply:
This paragraph explains how this works, without getting into the details. I guess we can use "generate" instead of "create" to underline that there's no need to create the project manually.

```suggestion
If you need to modify an application template, we recommend that you generate a real application based on this template and modify this application instead. This is easier because a real application can be run, and you can see how your modifications affect it, which would not be possible if you were to modify the template directly.
```